decentraland/components/presentational/MyAccountProgress.js

```diff
@@ -52,7 +53,7 @@ const styles = StyleSheet.create({
     justifyContent: "flex-start",
   },
   progress: {
-    marginTop: responsiveHeight(1),
+    marginTop: responsiveHeight(0),
```

Comment:
I suppose we could just omit this line instead of setting it to 0

Reply:
Sure.

Reply:
Extracted to: https://github.com/tasitlabs/tasit/issues/296

Reply:
Resolving this